### PR TITLE
update documentation on how to run all tests

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,24 +70,39 @@ The express server is available on at [http://localhost:8000](http://localhost:8
...
 `npm run eslint`
 
-### Test
+### Running Tests
 
-Tests are written in jest, also utilizing puppeteer for end-to-end tests.
-To run them `npm test`.
+Tests are written in jest, also utilizing puppeteer for end-to-end tests, and are run with `npm test`.
+
+To properly run all of the tests (including integration), you'll have to provide a couple environment variables,
```
Comment (jcoyne) [inline README.md]: missing "of"

File: README.md [hunk 2]
```diff
@@ -70,24 +70,39 @@ The express server is available on at [http://localhost:8000](http://localhost:8
 
 `npm run eslint`
 
-### Test
+### Running Tests
 
-Tests are written in jest, also utilizing puppeteer for end-to-end tests.
-To run them `npm test`.
+Tests are written in jest, also utilizing puppeteer for end-to-end tests, and are run with `npm test`.
```
Comment (jcoyne) [inline README.md]: I think this was clearer as two sentences.

File: README.md [hunk 3]
```diff
@@ -70,24 +70,39 @@ The express server is available on at [http://localhost:8000](http://localhost:8
...
 
-e.g.
+Putting it all together, to run all of the tests:
+
 ```sh
-COGNITO_TEST_USER_NAME='sinopia-devs+client-tester@lists.stanford.edu' COGNITO_TEST_USER_PASS='theActualPassword' npm run jest-cov
+COGNITO_TEST_USER_NAME='sinopia-devs+client-tester@lists.stanford.edu' COGNITO_TEST_USER_PASS='theActualPassword' npm test
 ```
 
+Note that if you have an instance of the dev server already running in a separate terminal, you may need to stop the server or you may get a port conflict
```
Comment (jcoyne) [inline README.md]: 👍

**Comment by peetucket** [comment]
updated and rebased

**Pull request merged**